### DISCUSSION TITLE
More labels (like `ubuntu-latest`)

### DISCUSSION THREAD

**Discussion by wxtim** [actions/virtual-environments#9014] [Ideas]
A lot of workflows that I maintain have matrices like:

```yaml
      matrix:
        ubuntu: ['ubuntu-20.04', 'ubuntu-22.04']
```

Could we have a label like `ubuntu-penultimate` or `ubuntu-latest-1` or even `ubuntu-earliest` so that we can have a fixed workflow:

```yaml
      matrix:
        ubuntu: ['ubuntu-latest-1', 'ubuntu-latest']
```

**Comment by erik-bershel** [comment]
Hey @wxtim! Such an interesting idea. We definitely didn’t have such plans. At first glance it seems that it will not be very popular. But after a little reflection, I can say that the idea has certain advantages.

**Reply by wxtim** [reply]
Thanks for getting back to me. I understand that it may well not be a priority, but I thought I'd plant the seed of an idea.

**Comment by EduPonz** [comment]
I would actually like to have a way to have a completely stable image. Things like #9491 happen way too often, so I'd like to have the ability to just find one that works and keep using that one without things changing under my feet.
